### ISSUE TITLE
Question about some details in the paper

### ISSUE THREAD

**Issue by yxliu-TAMU** [HParklab/IEFFEUM#1]
Dear Heechan Lee and Prof. Hahnbeom Park,

Thanks so much for your work, and well-maintained scripts. (By the way, the ΔG of MyUb with an A100 (40G) machine is 0.19)

1. In the paper, you mentioned that "Consequently, using this equilibrium ratio we calculated the distribution of pairwise distances between residues i and j in folded and unfolded states", is this distribution presented by the weighted average of the unfold and folded distogram?

2. To illustrate the importance of unfold ensemble, you compare the results with the IEFFEUM (baseline model). While training this baseline model, do you drop all equilibrium distogram prediction head directly? (so also no L_ensemble term in the loss function?)

3. In the final result, you also provide "per_resi_dG", how should I interpolate these numbers?

Thanks again for your great effort!

Best,
Yuxuan

**Comment by supergermy** [comment]
Thanks for asking, Yuxuan!

1. For every pair of residues, instead of a one-hot encoded (or RBF) distogram with a single weighted average value, the individual probability for the folded and unfolded state are used. (e.g., not [0,0,1,0] but [0,0.5,0.5,0] (btw, this is when Δ_G_=0))
2. Yes, for IEFFEUM, we dropped the equilibrium distogram prediction head (and thus L_ensemble). Leaving the head but setting L_ensemble to zero yielded similar results.
3. In short, the sum of per_resi_dG (a [N,] vector predicted for an N-length protein) is dG. We believe these values could be useful in many contexts, as demonstrated by our redesign of residues with negative per_resi_dG to enhance predicted ΔG.

If you have any other questions please feel free to ask!

Best regards,
Heechan
